dev-python/pymupdf/pymupdf-1.20.2.recipe

```diff
@@ -31,7 +31,7 @@ BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libmupdf$secondaryArchSuffix >= 1.20
 	devel:libfreetype$secondaryArchSuffix
-	swig
+	swig$secondaryArchSuffix
```

Comment:
There is no $secondaryArchSuffix for swig :)

https://github.com/haikuports/haikuports/blob/4f180bdb94db9749ae800820b271e262a12754aa/dev-lang/swig/swig-4.0.2.recipe#L33

Reply:
Strike that, my bad :)